### PR TITLE
User

### PR DESCRIPTION
**Pull request by wiliajc87**

### CODE REVIEW

**Comment by SStrausman** [comment]
Looks good to me.

File: app/controllers/services_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  def index
+    @services = Service.all
+  end
+
+  def new
+    @service = Service.new
+  end
+
+  def create
+    @service = Service.new(:service_params)
```
Comment (tjmasterson) [inline app/controllers/services_controller.rb]: I dont think this should be a symbol just "service_params"

**Comment by sowiecki** [comment]
There are merge conflict artifacts on Gemfile.lock @wiliajc87

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -3,10 +3,15 @@
 <head>
   <title>Pgtestphoto</title>
   <%= stylesheet_link_tag    'application', media: 'all' %>
+<<<<<<< HEAD
```
Comment (thewildpendulum) [inline app/views/layouts/application.html.erb]: Remove this merge conflict stuff

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: {
+  sessions: 'sessions'
```
Comment (thewildpendulum) [inline config/routes.rb]: Fix indentation

**Pull request merged**